node_modules/oae-core/createcollabdoc/js/createcollabdoc.js

```diff
@@ -110,8 +110,8 @@ define(['jquery', 'oae.core'], function ($, oae) {
                 // Update visibility for document
                 visibility = data.visibility;
 
-                managers = data.shared;
-                folders = data.folders;
+                managers = _.pluck(data.selectedPrincipalItems, 'shareId');
```

Comment:
Should `oae.setpermissions.changed.` just pass in the `shareId` array and the folder id `array?